static/app/components/events/highlights/highlightsIconSummary.tsx

```diff
@@ -58,10 +72,39 @@ export function HighlightsIconSummary({event, group}: HighlightsIconSummaryProps
   const releaseTag = event.tags?.find(tag => tag.key === 'release');
   const environmentTag = event.tags?.find(tag => tag.key === 'environment');
 
-  return items.length ? (
+  return items.length || screenshot ? (
     <Fragment>
       <IconBar>
-        <ScrollCarousel gap={3} aria-label={t('Icon highlights')}>
+        <ScrollCarousel gap={2} aria-label={t('Icon highlights')}>
+          {screenshot && (
+            <Fragment>
+              <ScreenshotButton
+                type="button"
+                borderless
+                size="zero"
+                icon={<IconAttachment color="subText" />}
+                title={t('View Screenshot')}
+                onClick={() => {
+                  const downloadUrl = `/api/0/projects/${organization.slug}/${group.project.slug}/events/${event.id}/attachments/${screenshot.id}/`;
+                  openModal(
+                    modalProps => (
+                      <ScreenshotModal
+                        {...modalProps}
+                        projectSlug={group.project.slug}
+                        eventAttachment={screenshot}
+                        downloadUrl={downloadUrl}
+                        attachments={attachments}
+                      />
+                    ),
+                    {modalCss}
+                  );
+                }}
+              >
+                <IconDescription>{t('Screenshot')}</IconDescription>
+              </ScreenshotButton>
+              {items.length > 1 && <Divider />}
```

Comment:
should we always have the divider here even if there is only one context? might be good to still have some kind of separation?

Reply:
maybe? it would be between the screenshot button and nothing. I assume most times the screenshot would appear other things would appear next to i t